packages/frontend2/src/server/features/tvl/get-tvl-projects.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  type BackendProject,
+  bridgeToBackendProject,
+  bridges,
+  getTvlAmountsConfig,
+  layer2ToBackendProject,
+  layer2s,
+  layer3ToBackendProject,
+  layer3s,
+} from '@l2beat/config'
+import { assert, type ProjectId, UnixTime } from '@l2beat/shared-pure'
+
+export interface TvlProject {
+  id: ProjectId
+  minTimestamp: UnixTime
+  type: BackendProject['type']
+  slug: string
+  sources: Map<
+    string,
+    {
+      name: string
+      minTimestamp: UnixTime
+    }
+  >
+}
+
+export function getTvlProjects(): TvlProject[] {
+  const projects = [
+    ...layer2s.map(layer2ToBackendProject),
+    ...layer3s.map(layer3ToBackendProject),
+    ...bridges.map(bridgeToBackendProject),
+  ]
+
+  const tvlAmounts = getTvlAmountsConfig(projects)
+
+  return projects.flatMap(({ projectId, type, slug }) => {
+    const amounts = tvlAmounts.filter((o) => o.project === projectId)
+    if (!amounts) {
+      return []
+    }
+    assert(amounts, 'Config not found: ' + projectId.toString())
```

Comment:
Will this case fall through? We already have truthy check above

Reply:
Leftover from the old code :) Removing!